packages/b/bandwhich/package.yml

```diff
@@ -17,4 +17,3 @@ build      : |
     cargo build --release    
 install    : |
     install -Dm00755 target/release/bandwhich $installdir/usr/bin/bandwhich
-    install -Dm00644 docs/bandwhich.1 -t $installdir/usr/share/man/man1/
```

Comment:
Why was this removed? The changelogs make it sound like the man.1 file should be available